agenta-backend/agenta_backend/routers/evaluation_router.py

```diff
@@ -52,11 +51,10 @@ async def fetch_evaluation_ids(
         List[str]: A list of evaluation ids.
     """
     try:
-        app = await db_manager.fetch_app_by_id(app_id=app_id)
         if isCloudEE():
             has_permission = await check_action_access(
                 user_uid=request.state.user_id,
-                project_id=str(app.project_id),
+                project_id=request.state.project_id,
```

Comment:
I wouldn't recommend this. Using `request.state.project_id` will make use of the logged-in user project and not the project that the evaluation belongs in. The reason why there's an `app_id` is because the frontend is yet to update its UI to pass the `project_id`.

Reply:
This has been worked on in _project checkpoints 4_. Awesome work, @jp-agenta !